requirements/base.txt

```diff
@@ -7,16 +7,17 @@ kornia==0.5.6
 lxml==4.6.5
 matplotlib==3.4.3
 networkx~=2.5
-numpy~=1.19.5
+# nncf==2.1.0
+numpy>=1.22.3
 omegaconf==2.1.1
+pandas==1.4.1
 opencv-python>=4.5.3.56
 opencv-contrib-python==4.5.5.62
-pandas~=1.1.5
 pillow==9.0.0
-pytorch-lightning==1.5.9
-torch==1.8.1
-torchvision==0.9.1
-torchtext==0.9.1
+pytorch-lightning==1.6.0
+torch==1.11.0
+torchvision==0.12.0
+torchtext==0.12.0
 scikit-image>=0.17.2
-scikit-learn==0.24.2
+scikit-learn==1.0.2
```

Comment:
it might be an idea to make these dependencies a bit more flexible. Rather than asking for the exact version, we could maybe do something like `scikit-learn>=0.24.2`. In this case would you be able to download the version you want? The reason is that there could be some usecases where another library depends on an earlier package version, but anomalib has the latest version as the dependency, which ultimately would causes hard package dependency conflicts.